support-frontend/assets/components/personalDetails/personalDetails.tsx

```diff
@@ -0,0 +1,122 @@
+// ----- Imports ----- //
+import { TextInput } from '@guardian/source-react-components';
+import type { ContributionType } from 'helpers/contributions';
+import { emailRegexPattern } from 'helpers/forms/formValidation';
+import type { UserTypeFromIdentityResponse } from 'helpers/identityApis';
+import { applyPersonalDetailsRules } from 'helpers/subscriptionsForms/rules';
+import { firstError } from 'helpers/subscriptionsForms/validation';
+import { classNameWithModifiers } from 'helpers/utilities/utilities';
+
+export type PersonalDetailsProps = {
+	email: string;
+	firstName: string;
+	lastName: string;
+	checkoutFormHasBeenSubmitted: boolean;
+	contributionType: ContributionType;
+	isSignedInPersonalDetails: boolean;
+	userTypeFromIdentityResponse: UserTypeFromIdentityResponse;
+	onEmailChange: (email: string) => void;
+	onFirstNameChange: (firstName: string) => void;
+	onLastNameChange: (lastName: string) => void;
+	signOutLink: React.ReactNode;
+	contributionState: React.ReactNode;
+};
+
+export function PersonalDetails({
+	email,
+	firstName,
+	lastName,
+	checkoutFormHasBeenSubmitted,
+	contributionType,
+	isSignedInPersonalDetails,
+	userTypeFromIdentityResponse,
+	onEmailChange,
+	onFirstNameChange,
+	onLastNameChange,
+	signOutLink,
+	contributionState,
+}: PersonalDetailsProps): JSX.Element {
+	const formErrors = applyPersonalDetailsRules({
+		firstName,
+		lastName,
+		email,
+		isSignedIn: isSignedInPersonalDetails,
+		userTypeFromIdentityResponse,
+	});
+
+	const getFormFieldError = (formField: string) =>
+		checkoutFormHasBeenSubmitted
+			? firstError(formField, formErrors)
+			: undefined;
```

Comment:
We'll be handling form validation through a different, Redux-based mechanism for the new checkout, so shouldn't need the error code in this component.

Reply:
Ah ok, do I pass any errors generated up to the container (eg TextInput error) and handle there? Is there an example anywhere I can have a look at?

Reply:
Actually, think understand, just remove all the associated error handling + props for the moment. Will do now, thanks

Reply:
Ok, all removed